web/client-api/src/main/java/io/deephaven/web/client/api/tree/JsTreeTable.java

```diff
@@ -209,455 +232,454 @@ public TreeRow(int offsetInSnapshot, Object[] dataColumns, Object rowStyleColumn
-            closed = true;
-            request.setIncludedOps(
-                    new TreeTableRequest.TreeRequestOperation[] {TreeTableRequest.TreeRequestOperation.Close});
-            request.setExpandedNodes(new TableDetails[0]);
-            request.setSorts(new SortDescriptor[0]);
-            request.setFilters(new FilterDescriptor[0]);
-            request.setColumns(new BitSet());
-            return request;
-        }
-
-        request.setExpandedNodes(
-                expandedMap.values().stream().map(TreeNodeState::toTableDetails).toArray(TableDetails[]::new));
-
-        final int hierarchicalChildrenColumnIndex = Arrays.stream(this.baseTable.getTableDef().getColumns())
-                .filter(col -> col.getName()
-                        .equals(this.baseTable.getTableDef().getAttributes().getTreeHierarchicalColumnName()))
-                .mapToInt(ColumnDefinition::getColumnIndex)
-                .findFirst()
-                .orElseThrow(() -> new IllegalStateException("TreeTable definition has no hierarchy column"));
-
-        request.setKeyColumn(hierarchicalChildrenColumnIndex);
-
-        // avoid sending filters unless they changed for smaller overhead on requests, there is no need to recompute
-        // filters on child tables, only on the root table
-        if (!filters.equals(nextFilters)) {
-            request.setFilters(
-                    nextFilters.stream().map(FilterCondition::makeDescriptor).toArray(FilterDescriptor[]::new));
-        } else {
-            request.setFilters(new FilterDescriptor[0]);
-        }
-
-        // always include the sort setup, the viewport content could have changed in practically any way
-        request.setSorts(nextSort.stream().map(Sort::makeDescriptor).toArray(SortDescriptor[]::new));
-
-        BitSet columnsBitset = baseTable.makeBitset(this.columns);
-        columnsBitset.set(hierarchicalChildrenColumnIndex);
-        request.setColumns(columnsBitset);
-        request.setViewportEnd((long) (double) lastRow);
-        request.setViewportStart((long) (double) firstRow);
-
-        request.setIncludedOps(nextRequestOps);
-        nextRequestOps = new TreeTableRequest.TreeRequestOperation[0];
-
-        return request;
-    }
-
-    private void enqueue(TreeTableRequest.TreeRequestOperation operation, JsRunnable r) {
-        if (queuedOperations != null) {
-            JsRunnable old = queuedOperations;
-            queuedOperations = () -> {
-                nextRequestOps[nextRequestOps.length] = operation;
-                old.run();
-                r.run();
-            };
-        } else {
-            queuedOperations = () -> {
-                nextRequestOps[nextRequestOps.length] = operation;
-                r.run();
-            };
+    private BitSet makeColumnSubscriptionBitset() {
+        // Build the bitset for the columns that are needed to get the data, style, and maintain structure
```

Comment:
Should be a JavaDoc:
```suggestion
    /**
     * Build the bitset for the columns that are needed to get the data, style, and maintain structure
     * @return BitSet for the columns
     */
    private BitSet makeColumnSubscriptionBitset() {
```